Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM microsoft/dotnet:2.2-sdk AS installer-env
+FROM mcr.microsoft.com/dotnet/core/sdk:2.2 AS installer-env
```

Comment:
what is the difference in microsoft vs mcr. Was there a change done by the team managing the base images?

Reply:
They are the same images, just hosted on Microsoft's own registry rather than Dockerhub. I believe the policy is to have all official microsoft images on mcr rather than dockerhub. The dotnet team moved their images there recently.

Reply:
sounds good, you may have to rerun appveyor build before merging.